SPECS-SIGNED/kernel-mshv-signed/kernel-mshv-signed.spec

```diff
@@ -0,0 +1,153 @@
+%global debug_package %{nil}
```

Comment:
Please also update the codeowners files so @microsoft/cbl-mariner-kata-containers are automatically added as reviewers for updates to these new specs